src/instructlab/configuration.py

```diff
@@ -173,9 +186,53 @@ def get_default_config():
             taxonomy_base=DEFAULT_TAXONOMY_BASE,
         ),
         serve=_serve(model_path=DEFAULT_MODEL_PATH),
+        train=_train(
+            train_args=TrainingArgs(
+                model_path=DEFAULT_MODEL_REPO,
+                data_path=DEFAULT_GENERATED_FILES_OUTPUT_DIR,
+                ckpt_output_dir=DEFAULT_CKPT_DIR,
+                data_output_dir=DEFAULT_OUT_DIR,
+                max_seq_len=4096,
+                max_batch_len=10000,
+                num_epochs=10,
+                effective_batch_size=3840,
+                save_samples=25000,
```

Comment:
This is way too high, set it to `250_000`

Reply:
It'll be `13.48G` per checkpoint, we save 2 checkpoints but we get rid of one set.